tests/integration.rs

```diff
@@ -0,0 +1,58 @@
+/* Checks we can verify (eg) netflix's partial cert chain. */
+extern crate webpki;
+extern crate ring;
+extern crate time;
+
+use ring::input::Input;
+use webpki::*;
+
+static ALL_SIGALGS: &'static [&'static SignatureAlgorithm] = &[
+  &ECDSA_P256_SHA1,
+  &ECDSA_P256_SHA256,
+  &ECDSA_P256_SHA384,
+  &ECDSA_P256_SHA512,
+  &ECDSA_P384_SHA1,
+  &ECDSA_P384_SHA256,
+  &ECDSA_P384_SHA384,
+  &ECDSA_P384_SHA512,
+  &RSA_PKCS1_2048_8192_SHA1,
+  &RSA_PKCS1_2048_8192_SHA256,
+  &RSA_PKCS1_2048_8192_SHA384,
+  &RSA_PKCS1_2048_8192_SHA512,
+  &RSA_PKCS1_3072_8192_SHA384
+];
+
+fn read_bytes(filename: &str) -> Vec<u8> {
+  use std::fs::File;
```

Comment:
Although I don't like it much, I decided to use the 4-space indent in the rest of the project since that seems to be the standard in Rust. So, please use four-space indent.